recipe/magento2.php

```diff
@@ -48,13 +54,14 @@
 // Tasks
 desc('Compile magento di');
 task('magento:compile', function () {
+    run('cd {{release_path}} && {{bin/composer}} dump-autoload -o');
```

Comment:
Any specific reason why this is in here?

Reply:
This problem should not occur on deploy - the generated folder will be recreated each deploy, leaving no stale classes behind.

Reply:
was impossible for me to deploy a m2.4 without dumping the autoload before di-compiling, got this error all the time
```
Warning: include(/var/www/html/vendor/composer/../../generated/code/Magento/Framework/App/ResourceConnection/Proxy.php):
failed to open stream: No such file or directory in /var/www/html/vendor/composer/ClassLoader.php on line 444
```
everything fine in m2.2 and m2.3

Reply:
Ok LGTM